vnext/Microsoft.ReactNative/Fabric/FabricUIManagerModule.cpp

```diff
@@ -53,7 +52,7 @@ FabicUIManagerProperty() noexcept {
 }
 
 FabricUIManager::FabricUIManager() {
-  facebook::react::CoreFeatures::enablePropIteratorSetter = true;
+  facebook::react::ReactNativeFeatureFlags::enableCppPropsIteratorSetter();
```

Comment:
remove this line, it is no longer going anything. 
But we do need to override the value of this flag in our feature flags - in ReactNativeWindowsFeatureFlags in ReactHost.cpp

Reply:
Removed - will we need to call the new override `ReactNativeWindowsFeatureFlags::enableCppPropsIteratorSetter()` in the constructor, in place of `ReactNativeFeatureFlags::enableCppPropsIteratorSetter()`?